v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java

```diff
@@ -106,6 +106,7 @@ public void processElement(ProcessContext c, MultiOutputReceiver output) {
           .get(SourceDbToSpannerConstants.ROW_TRANSFORMATION_SUCCESS)
           .output(RowContext.builder().setRow(sourceRow).setMutation(mutation).build());
     } catch (Exception e) {
+      LOG.error("Error while processing element", e);
```

Comment:
Do we want to keep this log statement?

Reply:
This is in exception path. 
Have you seen a lot of these in any of your test runs?

Unlike live template, we get to order foreign key constraints in Bulk, so in the most ideal (and correctly scaled - Dataflow, spanner etc) migration there should not be a ton of exceptions.